pkg/cmd/ai/configure.go

```diff
@@ -0,0 +1,34 @@
+package ai
+
+import (
+	"errors"
+
+	addCmd "github.com/buildkite/cli/v3/pkg/cmd/ai/add"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAIConfigure(f *factory.Factory) *cobra.Command {
+	var force bool
+
+	cmd := &cobra.Command{
+		Use:     "configure",
+		Aliases: []string{"config"},
```

Comment:
Ohh is this scoped to the parent command? Or will it clobber the existing "config" alias

Reply:
The scoping for an alias is to the parent of it, `bk ai config` == `bk ai configure`, `bk config` does not get overwritten, nor does it _allow_ `ai` config.